src/quo2/components/drawers/permission_context.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.drawers.permission-context
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]))
+
+(def radius 20)
```

Comment:
@ibrkhalil I've been thinking about these concepts lately because they've been coming up related to the discussion around `^private`.

I'm not sure what, if anything, constitutes global in clojure/script (maybe namespaces themselves?), but it's not like javascript where you have global variables that you have to worry about.

Clojure will always intern a var in a namespace, to my knowledge. I think you won't ever find a var that is not in a namespace. [interning](https://clojure.org/reference/vars#interning)